build.gradle

```diff
@@ -170,7 +183,11 @@ allprojects {
 }
 
 subprojects {
-    task dependencyCheckAnalyze(type: org.owasp.dependencycheck.gradle.tasks.Analyze)
+    task dependencyCheckAnalyze(type: org.owasp.dependencycheck.gradle.tasks.Analyze) {
+        // disable analyze in cas-server-core-api and cas-server-core because
+        // support and webapp subprojects will include their dependencies transitively
+        enabled = !"${project.name}".contains("cas-server-core")
```

Comment:
Please change individual line-comments to multi-lines using the `/* whatever */` syntax.